cmd/envtool/setup.go

```diff
@@ -68,6 +74,92 @@ func setupMongoDB(ctx context.Context, logger *slog.Logger, uri, name string) er
 	return ctx.Err()
 }
 
+// setupYugabyteDB configures YugabyteDB container by creating a user.
+// The user created upon Docker container initialization cannot authenticate with FerretDB.
+// It waits for DocumentDB extension to be created before creating the user.
+func setupYugabyteDB(ctx context.Context, uri string, l *slog.Logger) error {
```

Comment:
Let's make it a generic function for all PostgreSQL-like backends, and use it for PostgreSQL too, as discussed